x-pack/auditbeat/module/system/socket/template.go

```diff
@@ -35,7 +35,7 @@ var baseTemplateVars = common.MapStr{
 // These functions names vary between kernel versions. The first available one
 // will be selected during setup.
 var functionAlternatives = map[string][]string{
-	"IP_LOCAL_OUT":      {"ip_local_out", "ip_local_out_sk"},
+	"IP_LOCAL_OUT":      {"ip_local_out_sk", "__ip_local_out", "ip_local_out"},
```

Comment:
are there going to be implications in inverting the precedence? Why did we do `ip_local_out_sk` second previously?

Reply:
Needed to change the precedence because some old kernels that need `ip_local_out_sk` have an incompatible `__ip_local_out`.

Tested in all my VMs in different kernels 2.x / 3.x / 4.x / 5.x. I really need to add some proper automated tests like in beats_tester to ensure things don't break.